FinnUI/Sources/Fullscreen/MotorTransactionInsurancePickerView/MotorTransactionInsuranceView.swift

```diff
@@ -83,6 +85,11 @@ class MotorTransactionInsuranceView: UIView {
             logoImageView.setImage(fallbackImage, animated: false)
         }
     }
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        logoImageView.layer.borderColor = .logoImageBorderColor
```

Comment:
you already figured it out, but the UIColor resolves dynamically when the trait collection is available, like in these viewController methods, since you calculate a cgColor from the dynamic UIColor, if done somewhere when the `traitCollection` is not set, you will get the light value by default